mwsql-blogpost.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>I'm not sure 'future of mwsql' is the right heading for this section. Maybe 'constraints to remember'?</p><p>Small sentence edit:</p><blockquote>This is less of a problem in a WMF-hosted environment, such as PAWS [todo: add link], where the dumps are available through a public directory. Having the opportunity to inspect a file before committing to download all of it, as well as being able to process it <em>as</em> it is downloading (streaming), would be a huge performance improvement for users working in non-WMF environments.</blockquote><p> </p><p>Change "project info" heading to "mwsql project info".</p><p>Fix missing link ("features(link)").</p><p>Instead of saying "get in touch" provide a specific communication channel, like "comment on this blog post" or "post on this Talk page".</p><p> </p><p>Overall: really great job on this, it's excellent!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/blancadesal/wm-tutorials-and-blogposts/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>I decided to keep 'future of mwsql as a heading', mostly for the lack of a better idea, but have made changes according to your suggestions otherwise. </p>